tensornetwork/backends/numpy/numpy_backend_test.py

```diff
@@ -123,3 +123,106 @@ def test_convert_bad_test():
   backend = numpy_backend.NumPyBackend()
   with pytest.raises(ValueError):
     backend.convert_to_tensor(tf.ones((2, 2)))
+
+
+def test_norm():
+  backend = numpy_backend.NumPyBackend()
+  a = backend.convert_to_tensor(np.ones((2, 2)))
+  assert backend.norm(a) == 2
+
+
+def test_eye():
+  backend = numpy_backend.NumPyBackend()
+  a = backend.eye(N=4, M=5)
+  np.testing.assert_allclose(np.eye(4, 5), a)
+
+
+def test_ones():
+  backend = numpy_backend.NumPyBackend()
+  a = backend.ones((4, 4))
+  np.testing.assert_allclose(np.ones((4, 4)), a)
+
+
+def test_zeros():
+  backend = numpy_backend.NumPyBackend()
+  a = backend.zeros((4, 4))
+  np.testing.assert_allclose(np.zeros((4, 4)), a)
+
+
+def test_randn():
+  backend = numpy_backend.NumPyBackend()
+  a = backend.randn((4, 4))
+  assert a.shape == (4, 4)
+
+
+def test_eye_dtype():
+  backend = numpy_backend.NumPyBackend(dtype=np.float64)
+  dtype = np.float32
+  a = backend.eye(N=4, M=4, dtype=dtype)
+  assert a.dtype == dtype
+
+
+def test_ones_dtype():
+  backend = numpy_backend.NumPyBackend(dtype=np.float64)
+  dtype = np.float32
```

Comment:
Please parameterize the dtype so we make sure it works for all of the supported values.